### PR TITLE
first 6 problems - arrays/strings

### PR DESCRIPTION
**Pull request by Svetlana-Leonova**

### CODE REVIEW

**Comment by sphinx20** [comment]
1. Solve two sum problem for a sorted algo as well. If the interviewer sees you solving this one quick they are definitely going to ask you that question next. Also learning 2 pointers is a good idea because interviewers love it.
2. Valid Palindrome is missing.

File: Svetlana/README.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+let currentNum = nums[i];
+let difference = target - currentNum;
+if (difference in map) {
+result.push(i, map[difference])
+} else {
+map[currentNum] = i;
+}
+}
+return result;
+};
```
Comment (apaniagua6) [inline Svetlana/README.md:16]: Please wrap your code in proper markdown so it is easier to read for your future self: 

https://github.com/CheetCodes/Cheetcode-Study-Prep-March-2021/blob/main/Jenna/README.md

File: Svetlana/README.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+const area = length \* lowerWall;
+maxArea = Math.max(area, maxArea)
+if (height[left] < height[right]) {
+left++;
+} else {
+right--;
+}
+}
+return maxArea
+};
```
Comment (apaniagua6) [inline Svetlana/README.md:36]: Solid naming! 
lowerWall =
length = 
area =

File: Svetlana/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+if (height[rightPointer] >= maxRight) {
+maxRight = height[rightPointer];
+} else {
+totalWater += maxRight - height[rightPointer]
+}
+rightPointer--;
+}
+}
+return totalWater;
+};
```
Comment (apaniagua6) [inline Svetlana/README.md:64]: This is a hard problem. we will revisit various variations of this but lets not spend too much time on this one for now.

File: Svetlana/README.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+tCharsToDelete += 2;
+while (tCharsToDelete > 0) {
+tCharsToDelete--;
+tPointer--;
+if (T[tPointer] === "#") {
+tCharsToDelete+=2;
+}
+}
+}
+if (S[sPointer] !== T[tPointer]) {
```
Comment (apaniagua6) [inline Svetlana/README.md:94]: Good stuff! Please add the time and space complexities of the brute force and and optimal solutions 
Time O(N), Space(N) vs Time O(N), Space (1)

**Review by apaniagua6 — APPROVED** [review]

**Pull request merged**